src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -26,72 +26,72 @@ CascadiaSettings::~CascadiaSettings()
 
 ColorScheme _CreateCampbellScheme()
 {
-    ColorScheme campbellScheme { L"Campbell",
-                                 RGB(242, 242, 242),
-                                 RGB(12, 12, 12) };
-    auto& campbellTable = campbellScheme.GetTable();
-    auto campbellSpan = gsl::span<COLORREF>(&campbellTable[0], gsl::narrow<ptrdiff_t>(COLOR_TABLE_SIZE));
-    Microsoft::Console::Utils::InitializeCampbellColorTable(campbellSpan);
-    Microsoft::Console::Utils::SetColorTableAlpha(campbellSpan, 0xff);
-
-    return campbellScheme;
+	ColorScheme campbellScheme{ L"Campbell",
+								 RGB(242, 242, 242),
+								 RGB(12, 12, 12) };
+	auto& campbellTable = campbellScheme.GetTable();
+	auto campbellSpan = gsl::span<COLORREF>(&campbellTable[0], gsl::narrow<ptrdiff_t>(COLOR_TABLE_SIZE));
+	Microsoft::Console::Utils::InitializeCampbellColorTable(campbellSpan);
+	Microsoft::Console::Utils::SetColorTableAlpha(campbellSpan, 0xff);
+
+	return campbellScheme;
 }
 
 ColorScheme _CreateSolarizedDarkScheme()
 {
 
-    ColorScheme solarizedDarkScheme { L"Solarized Dark",
-                                      RGB(253, 246, 227),
-                                      RGB(  7, 54,  66) };
-    auto& solarizedDarkTable = solarizedDarkScheme.GetTable();
-    auto solarizedDarkSpan = gsl::span<COLORREF>(&solarizedDarkTable[0], gsl::narrow<ptrdiff_t>(COLOR_TABLE_SIZE));
-    solarizedDarkTable[0]  = RGB(  7, 54, 66);
-    solarizedDarkTable[1]  = RGB(211, 1, 2);
-    solarizedDarkTable[2]  = RGB(133, 153, 0);
-    solarizedDarkTable[3]  = RGB(181, 137, 0);
-    solarizedDarkTable[4]  = RGB( 38, 139, 210);
-    solarizedDarkTable[5]  = RGB(211, 54, 130);
-    solarizedDarkTable[6]  = RGB( 42, 161, 152);
-    solarizedDarkTable[7]  = RGB(238, 232, 213);
-    solarizedDarkTable[8]  = RGB(  0, 43, 54);
-    solarizedDarkTable[9]  = RGB(203, 75, 22);
-    solarizedDarkTable[10] = RGB( 88, 110, 117);
-    solarizedDarkTable[11] = RGB(101, 123, 131);
-    solarizedDarkTable[12] = RGB(131, 148, 150);
-    solarizedDarkTable[13] = RGB(108, 113, 196);
-    solarizedDarkTable[14] = RGB(147, 161, 161);
-    solarizedDarkTable[15] = RGB(253, 246, 227);
-    Microsoft::Console::Utils::SetColorTableAlpha(solarizedDarkSpan, 0xff);
-
-    return solarizedDarkScheme;
+	ColorScheme solarizedDarkScheme{ L"Solarized Dark",
+									  RGB(253, 246, 227),
+									  RGB(7, 54,  66) };
+	auto& solarizedDarkTable = solarizedDarkScheme.GetTable();
+	auto solarizedDarkSpan = gsl::span<COLORREF>(&solarizedDarkTable[0], gsl::narrow<ptrdiff_t>(COLOR_TABLE_SIZE));
+	solarizedDarkTable[0] = RGB(7, 54, 66);
+	solarizedDarkTable[1] = RGB(211, 1, 2);
+	solarizedDarkTable[2] = RGB(133, 153, 0);
+	solarizedDarkTable[3] = RGB(181, 137, 0);
+	solarizedDarkTable[4] = RGB(38, 139, 210);
+	solarizedDarkTable[5] = RGB(211, 54, 130);

```

Comment:
Also, please back out unchanged portions (I bet this'll mostly go away after tabs switched back to spaces)